lib/temporal/client/grpc_client.rb

```diff
@@ -311,7 +311,7 @@ def terminate_workflow_execution(
             run_id: run_id,
           ),
           reason: reason,
-          details: to_details_payload(details)
+          details: to_details_payloads(details)
```

Comment:
Trivial bug fix for the recent serialization change.  It's tested in the new terminate integration test.